Source/VisualRecognitionV3/VisualRecognition.swift

```diff
@@ -608,7 +607,398 @@ public class VisualRecognition {
             case .failure(let error): failure?(error)
             }
         }
+    }
+    
+    /** 
+     Create a new collection. A maximum of five collections can be created.
+     
+     - parameter name:  The name of the new collection. The name can be a maximum of 128 UTF-8
+        characters. The name cannot contain any spaces.
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the newly created collection.
+    */
+    public func createCollection (
+        collectionName name: String,
```

Comment:
Let's try to match up the parameter names with the parameter names we chose in other services, so we have a more uniform naming standard :)